actions/run-stage/action.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: Run a MPyL stage
```

Comment:
won't the stage always be `deploy`?

Reply:
Not at the moment, it's [whatever stage is passed as an input](https://github.com/Vandebron/gh-mpyl/pull/26/files/bf43f6b4e8e886bf9e6195486bd737e5336194a5#diff-d461c0a5101f381ca78bcccca24295cdf26cf624fcf61d1788bb0bfb6f4098b6R8-R10).